lms/djangoapps/grades/tasks.py

```diff
@@ -91,9 +90,6 @@ def _recalculate_subsection_grade(self, **kwargs):
     """
     try:
         course_key = CourseLocator.from_string(kwargs['course_id'])
-        if not PersistentGradesEnabledFlag.feature_enabled(course_key):
-            return
```

Comment:
It should have no effect on edx.org since it's already enabled for all courses, right?